src/applications/mhv-medications/containers/PrescriptionDetailsDocumentation.jsx

```diff
@@ -170,6 +171,7 @@ const PrescriptionDetailsDocumentation = () => {
       if (!isLoadingDoc && !hasDocApiError && !isLoadingRx && htmlContent) {
         contentRef.current.innerHTML = htmlContent ?? '';
       }
+      focusElement(document.querySelector('h1'));
```

Comment:
should it focus h1 every time the below properties change or only on load?

Reply:
Can you please double check with Bobby if that's okay - to focus h1 after information gets updated?

Reply:
well the thing is that the the h1 wont even appear until html content is loaded so I can have the focus set on the h1 after `isLoadingDoc, hasDocApiError, isLoadingRx,` and it wont matter since the h1 will only load once the html content is loaded, so its dependent on the html content(see line 204). Given that if we want to separate the focus from the html content then we would have to make a condition for the h1 to load from redux memory rather than it being dependent on the `htmlContent` and then have the `htmlContent` load separately. Would you recommend this re-structuring of the jsx and useEffect instead?

Reply:
Gotcha. I am ok with that as long as the focus won't jump when you let's say download the PDF?

Reply:
link to slack thread for video showing a pdf download demo for the context of this h1. https://dsva.slack.com/archives/C059Q28DV99/p1726515951171939